pkg/kubelet/kubelet.go

```diff
@@ -2388,13 +2388,17 @@ func (kl *Kubelet) isOutOfDisk() bool {
 	// Check disk space once globally and reject or accept all new pods.
 	withinBounds, err := kl.diskSpaceManager.IsRuntimeDiskSpaceAvailable()
 	// Assume enough space in case of errors.
-	if err == nil && !withinBounds {
+	if err != nil {
+		glog.Errorf("failed to check if disk space is available for the runtime: %v", err)
```

Comment:
A question beyond this PR: When kubelet cannot check the disk space, it'd just admit the pods. That doesn't sound desirable to me... It seems reasonable to fail in this case. WDYT?